packages/contracts/src/dollar/core/UbiquityCreditToken.sol

```diff
@@ -5,8 +5,9 @@ import {ManagerFacet} from "../facets/ManagerFacet.sol";
 import {ERC20Ubiquity} from "./ERC20Ubiquity.sol";
 import {IERC20Ubiquity} from "../../dollar/interfaces/IERC20Ubiquity.sol";
 import "../libraries/Constants.sol";
+import {IUAR} from "../../ubiquistick/interfaces/IUAR.sol";
 
-contract UbiquityCreditToken is ERC20Ubiquity {
+contract UbiquityCreditToken is ERC20Ubiquity, IUAR {
```

Comment:
`IUAR` is somewhat deprecated, pls remove the inheritance

Reply:
remove the import https://github.com/ubiquity/ubiquity-dollar/blob/21f0514176801209f71e85ed8e6cbfef4f36e385/packages/contracts/src/ubiquistick/SimpleBond.sol#L10 @AnakinSkywalkeer

Reply:
> remove the import
> 
> https://github.com/ubiquity/ubiquity-dollar/blob/21f0514176801209f71e85ed8e6cbfef4f36e385/packages/contracts/src/ubiquistick/SimpleBond.sol#L10
> 
> @AnakinSkywalkeer

this `IUAR` is used [here](https://github.com/ubiquity/ubiquity-dollar/blob/21f0514176801209f71e85ed8e6cbfef4f36e385/packages/contracts/src/ubiquistick/SimpleBond.sol#L188)